src/components/InstantSearch.vue

```diff
@@ -9,7 +9,7 @@
 import instantsearch from 'instantsearch.js/es/';
 import { createSuitMixin } from '../mixins/suit';
 import { warn } from '../util/warn';
-import { addAlgoliaAgent } from '../util/client';
+import { version } from '../../package.json'; // rollup does pick only what needed from json
```

Comment:
It would be better to not rely on JSON but rather relying on a plain JS file. We can drop the dependency on the JSON Rollup plugin then. But I'm fine if it's stays like this though.

Reply:
I wanted to avoid adding another build step in the release process. IMO if it would show any problems at some point, we can still switch at that point.